test/TvmClient.test.js

```diff
@@ -207,53 +207,49 @@ describe('getAzureBlobCredentials', () => {
       expect(mockLogError).toHaveBeenCalledWith(expect.stringContaining(fakeTVMInput.ow.namespace))
       expect(mockLogError).toHaveBeenCalledWith(expect.not.stringContaining(fakeTVMInput.ow.auth))
     })
-    test('when tvm response has a server error with longer timeout', async () => {
-      jest.setTimeout(30000)
+    test('when tvm response has a server error with default maxRetries and initialDelayInMillis', async () => {
       // fake the fetch to the TVM
       fetch.mockResolvedValue(wrapInFetchError(500))
       fakeTVMInput.cacheFile = false
       const start = new Date()
       const tvmClient = await TvmClient.init(fakeTVMInput)
       await expect(tvmClient.getAzureBlobCredentials.bind(tvmClient)).toThrowStatusError(500)
-      expect(new Date() - start).toBeGreaterThan(3000)
+      // Retry delays would be 100 + 200 + 400 = 700
+      expect(new Date() - start).toBeGreaterThan(690)
```

Comment:
same comment for all tests: do not add wait time in tests, we should mock the timeout function
https://jestjs.io/docs/en/timer-mocks.html